core/kazoo_config/src/kapps_config_util.erl

```diff
@@ -49,9 +49,9 @@ load_config_from_reseller(Account, Config) ->
     ResellerId = kz_services:find_reseller_id(Account),
     load_config_from_account(ResellerId, Config).
 
--spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()} | {error, any()}.
+-spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()}.
 load_config_from_system(_Account, Config) ->
-    {'ok', kz_json:get_value(<<"default">>, maybe_new(kapps_config:get_category(Config)))}.
+    {'ok', kz_json:get_value(<<"default">>, kapps_config:get_category(Config), kz_json:new())}.
```

Comment:
i think the crash is not intended. i'm trying to figure out what's missing here.
were the schemas generated with `make apis` ? asking this because
get_config that you're calling above calls `load_default_config`, so if the schema exists and is loaded in system_schemas it should return something not `undefined`.